lms/services/lti_grading/_v13.py

```diff
@@ -95,7 +95,7 @@ def get_score_maximum(self, resource_link_id) -> float | None:
     def sync_grade(  # noqa: PLR0913
         self,
         application_instance: ApplicationInstance,
-        lis_outcome_service_url: str,
+        assignment: Assignment,
```

Comment:
Accommodate the change in the interface for 1.3. Take the URL from the assignment.
T